build/scripts/cache_projects.sh

```diff
@@ -24,14 +24,17 @@ TEMP_REPO="${TEMP_DIR}/cloned"
 # Clone a git repository and create an archive zip at a specified location
 # Args:
 #   $1 - URL of git repo
-#   $2 - (optional) branch to archive
+#   $2 - branch to archive
 #   $3 - destination path for the archived project zip file
 function cache_project() {
   local repo="$1"
   local branch="$2"
   local destination="$3"
-  git clone "$repo" -b "$branch" --depth 1 "$TEMP_REPO" &>/dev/null
-  git archive "$branch" --remote="$TEMP_REPO" --format zip --output "$destination"
+  rm -fr "$TEMP_REPO"
+  git clone "$repo" -b "$branch" --depth 1 "$TEMP_REPO" -q
+  pushd "$TEMP_REPO" &>/dev/null
+    git archive "$branch" -o "$destination"
```

Comment:
Should this command be in line with the other ones?

Reply:
Could be, but that block is "pushd" in. ;)

(Personal style choice. Helps me remember when I'm in a pushd/popd block.)

Reply:
Ah gotcha!, that makes sense!